src/utils/storage.ts

```diff
@@ -0,0 +1,28 @@
+import {AsyncStorage} from 'react-native';
+
+export async function getItem(key: string) {
+  try {
+    const val = await AsyncStorage.getItem(key);
+    return val;
+  } catch (error) {
+    console.log(`get ${key} value failed!`);
+  }
+}
+
+export async function saveItem(key: string, value: any) {
+  try {
+    const val = await AsyncStorage.setItem(key, value);
```

Comment:
Please check `value` before set, because if you pass `null` as a value you get native crash here!